src/main/java/com/crud/library/domain/TitleQuantityDto.java

```diff
@@ -10,7 +10,6 @@
 @NoArgsConstructor
 @AllArgsConstructor
 public class TitleQuantityDto {
-
```

Comment:
czym jest quantity? Czy ta encja na pewno jest potrzebna?

Reply:
liczba ta to ilość kopii danego tytułu. Jednak poprawiłem nazewnictwo